app/pages/home-not-logged-in.jsx

```diff
@@ -89,6 +92,21 @@ export default class HomePage extends React.Component {
     );
   }
 
+  getPromotedProjects() {
+    apiClient.type('projects').get({ id: Object.keys(FEATURED_PROJECTS), cards: true })
+    .then((projects) => {
+      Promise.all(projects.map((project) => {
```

Comment:
I don't think this needs to be a `Promise.all`, you can just return `projects.map()...` as there's nothing async in here